features/user/managing_users/deleting_account.feature

```diff
@@ -5,7 +5,8 @@ Feature: Deleting the customer account
     I want to be able to remove customer account details from the system
 
     Background:
-        Given there is a user "theodore@example.com" identified by "pswd"
+        Given the store has customer "customer@example.com"
```

Comment:
We generate CSRF token based on customer ID instead of user ID. Before this change, user ID and customer ID were the same, so it was passing.